bin/i18n/resources/dashboard/text_to_speech/sync_out.rb

```diff
@@ -42,8 +42,16 @@ def tts_units
             @tts_units ||= Unit.tts_enabled
           end
 
+          def mutex
+            @mutex ||= Mutex.new
+          end
+
           def sanitize_tts(text)
-            ::TextToSpeech.sanitize(text || '')
+            # Because Redcarpet instance is not thread-safe (https://github.com/vmg/redcarpet/issues/570),
+            # sharing the Markdown formatter in multi-thread operations may cause the error:
```

Comment:
This is a great comment, thank you! I expect this will be super useful to maintainers in several years' time.

Two super tiny grammar nits, mostly just because I know you appreciate them:

```suggestion
            # Because the Redcarpet instance is not thread-safe (https://github.com/vmg/redcarpet/issues/570),
            # sharing the Markdown formatter in multi-threaded operations may cause the error:
```

Reply:
Thank you, Elijah! I do appreciate them very much!